static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video-v2.js

```diff
@@ -0,0 +1,100 @@
+define([
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/_',
+    'common/utils/$',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/creatives/expandable-video-v2.html'
+], function (
+    bean,
+    bonzo,
+    fastdom,
+    _,
+    $,
+	detect,
+    mediator,
+    storage,
+    template,
+    svgs,
+    ExpandableVideoTpl
+) {
+
+    /**
+     * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10028247
```

Comment:
What exactly is this comment? Is this the creative as it's uploaded to DFP itself?